spring-cloud-aws-parameter-store-config/src/test/java/io/awspring/cloud/paramstore/AwsParamStorePropertiesTest.java

```diff
@@ -78,6 +79,18 @@ void acceptsBackslashAsProfileSeparator() {
 		assertThatNoException().isThrownBy(properties::afterPropertiesSet);
 	}
 
+	@Test
+	void checkExceptionLoggingForPrefix() {
+		try {
```

Comment:
better to use `assertThatThrownBy()` from AssertJ. 

Here, if exception wouldn't be thrown, the test would pass.